SPECS/net-snmp/net-snmp.spec

```diff
@@ -86,38 +90,57 @@ rm -rf %{buildroot}/*
 
 %files devel
 %defattr(-,root,root)
+%{_datadir}
 %{_includedir}
 %{_libdir}/*.la
-%{_libdir}/perl5
 %{_libdir}/*.so
-%{_datadir}
-%exclude /usr/lib/perl5/*/*/perllocal.pod
+%{_libdir}/perl5
+%{_libdir}/pkgconfig/*.pc
+%exclude %{_lib}/perl5/*/*/perllocal.pod
 
 %changelog
+* Mon Oct 12 2020 Joe Schmitt <joschmit@microsoft.com> - 5.9-1
```

Comment:
I'm flattered but I think this is a bad copy-paste :)

Reply:
Nah, this is correct - I just wanted to blame you in case of any issues. I'll fix this in a second...:)